packages/date-helpers/src/const.ts

```diff
@@ -7,3 +7,6 @@ export const DATETIME_FORMAT_STRING = "yyyy-MM-dd HH:mm:ss";
 export const FAR_FUTURE_DATE = "2999-12-31";
 export const FAR_PAST_DATE = "2000-01-01";
 export const PAST_DATE = "2020-01-01";
+
+// Maximum possible date for JS
+export const MAX_DATE = new Date(8640000000000000);
```

Comment:
Lint 🤔

Reply:
I'm sorry, what about linting here? :sweat_smile:

Reply:
Looks like it couldnt tell it was being used as a sub-import. Added it to the barrel to keep eslint happy :woman_shrugging: 

[add const to barrel to appease linter](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13290/commits/bc3da5b545f2ce60385d818508fe74bec8dc80f7)